src/twisted/web/resource.py

```diff
@@ -240,7 +243,10 @@ def render(self, request):
 
         @see: L{IResource.render}
         """
-        m = getattr(self, 'render_' + nativeString(request.method), None)
+        method = u'render_' + request.method.decode("ascii")
+        if not isinstance(method, str):
```

Comment:
Am I mistaken, if method happens to be unicode, this will raise an AttributeError where before nativeString() would have just done the right thing?

Reply:
No, you are mistaken.  With Python 2.7, getattr() works with a unicode argument.
On Python 3, getattr() will fail with a bytes argument.  So it is safe to use getattr() with a unicode arg on Python 2.7 and Python 3.

Reply:
So I am, thank you for the clarification!